utils/frontend/utils/Driver.ts

```diff
@@ -27,7 +27,7 @@ export const DriverBuilder = (function () {
 
     let caps: Capabilities = new Capabilities();
     caps = Capabilities.chrome();
-    caps.set("version", "110.0");
+    caps.set("version", "112.0");
```

Comment:
perhaps the version should come from an env. 
looks like tedious tasks updating chomeDriver, isn't it?

Reply:
true! will improve in next PR